sdk/src/main/java/live/talkshop/sdk/utils/networking/URLs.kt

```diff
@@ -7,19 +7,22 @@ import live.talkshop.sdk.utils.helpers.HelperFunctions.isNotEmptyOrNull
 object URLs {
     private const val URL_BASE = "https://staging.cms.talkshop.live/"
     private const val URL_CC_BASE = "https://assets.talkshop.live/events/"
+    private const val URL_ASSET_BASE_PROD = "https://assets-dev.talkshop.live/"
+    private const val URL_ASSET_BASE_STAGING = "https://assets.talkshop.live/"
```

Comment:
`assets-dev` is staging url and the other one is Production url

Reply:
Fixed!